_posts/2018-01-18-From-Manual-to-Automatic.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+authors: Hugo Fernandes, José Ferreira, Sofia Gomes, Elaine Navalho, João Ventura
+title: From Manual to Automatic
+---
+
+In the more recent years the interest on test automation has been increased significantly. Some of the reasons behind this growth include a greater need for faster software delivery processes, increased productivity and cost/effort reduction associated to the tests itself. This article will focus on test automation, its advantages and drawbacks and when and how should the transition from manual to automatic be made. 
+The article will also provide some examples of test automation tools, programming languages used for test automation and continuous integration software.
+
+## How vital is test automation in today’s world. 
+
+We live in a digital age. Every day new applications are deployed in productions systems and product features are improved, adjusted and updated with the objective of promoting the same products in an increasingly competitive market.
```

Comment:
`productions systems` should be `production systems`

Reply:
Done